modules/internal/ChapelArray.chpl

```diff
@@ -2395,10 +2465,14 @@ module ChapelArray {
   // integers and ranges; that is, it is a valid argument list for rank
   // change
   proc _validRankChangeArgs(args, type idxType) param {
+    proc _isRange(type idxType, r: range(?)) param return true;
```

Comment:
Why _validRankChangeArgs changed? Need at least an explanation in the PR message.